Bottom: f4ae5338aa1b7a8a0e3dc8e556f8e8fba2911c9b
Top:    49dbc9f734d7036f4d8e96c556c03c701446218c
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-11-18 20:59:27 +0000

Refresh of attempt-lesstif-gl-support

---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index fecd844..7b99b26 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -4450,7 +4450,6 @@ lesstif_finish_debug_draw (void)
    */
 }
 
-
 #include "dolists.h"
 
 void
